sbb_newspapers_org_image/eynollah.py

```diff
@@ -1,306 +1,198 @@
-import xml.etree.ElementTree as ET
-import warnings
-import click
-import time
-from multiprocessing import Process, Queue, cpu_count
+from shapely import geometry
+from lxml import etree as ET
 from matplotlib import pyplot, transforms
 import matplotlib.patches as mpatches
 import imutils
 
-warnings.filterwarnings('ignore')
-#
-__doc__ = \
-    """
-    tool to extract table form data from alto xml data
-    """
+from .utils.contour import (
+    contours_in_same_horizon,
+    filter_contours_area_of_image_interiors,
+    filter_contours_area_of_image_tables,
+    filter_contours_area_of_image,
+    find_contours_mean_y_diff,
+    find_features_of_contours,
+    find_new_features_of_contoures,
+    get_text_region_boxes_by_given_contours,
+    get_textregion_contours_in_org_image,
+    return_bonding_box_of_contours,
+    return_contours_of_image,
+    return_contours_of_interested_region,
+    return_contours_of_interested_region_and_bounding_box,
+    return_contours_of_interested_region_by_min_size,
+    return_contours_of_interested_textline,
+    return_parent_contours,
+    return_contours_of_interested_region_by_size,
+)
+
+from .utils.rotate import (
+    rotate_image,
+    rotate_max_area,
+    rotate_max_area_new,
+    rotatedRectWithMaxArea,
+    rotation_image_new,
+    rotation_not_90_func,
+    rotation_not_90_func_full_layout,
+    rotyate_image_different,
+)
+
+from .utils.separate_lines import (
+    seperate_lines,
+    seperate_lines_new_inside_teils,
+    seperate_lines_new_inside_teils2,
+    seperate_lines_vertical,
+    seperate_lines_vertical_cont,
+    textline_contours_postprocessing,
+    seperate_lines_new2,
+    return_deskew_slop,
+)
+
+from .utils.drop_capitals import (
+    adhere_drop_capital_region_into_cprresponding_textline,
+    filter_small_drop_capitals_from_no_patch_layout
+)
+
+from .utils.marginals import get_marginals
+
+from .utils.resize import resize_image
+
+from .utils import (
+    boosting_headers_by_longshot_region_segmentation,
+    crop_image_inside_box,
+    find_features_of_lines,
+    find_num_col,
+    find_num_col_by_vertical_lines,
+    find_num_col_deskew,
+    find_num_col_only_image,
+    isNaN,
+    otsu_copy,
+    otsu_copy_binary,
+    return_hor_spliter_by_index_for_without_verticals,
+    delete_seperator_around,
+    return_regions_without_seperators,
+    put_drop_out_from_only_drop_model,
+    putt_bb_of_drop_capitals_of_model_in_patches_in_layout,
+    check_any_text_region_in_model_one_is_main_or_header,
+    small_textlines_to_parent_adherence2,
+    order_and_id_of_texts,
+    order_of_regions,
+    implent_law_head_main_not_parallel,
+    return_hor_spliter_by_index,
+    combine_hor_lines_and_delete_cross_points_and_get_lines_features_back_new,
+    return_points_with_boundies,
+    find_number_of_columns_in_document,
+    return_boxes_of_images_by_order_of_reading_new,
+)
+
+
+SLOPE_THRESHOLD = 0.13
+VERY_LARGE_NUMBER = 1000000000000000000000
```

Comment:
I suppose @vahidrezanezhad can explain what the `VERY_LARGE_NUMBER` (`res me`) is used for?

Reply:
I introduced that constant to reduce the amount of "magic numbers" in the code. But checking again now, I see that it is not actually used in live code, only in `unused,py`. I'll remove it.

Reply:
4d8a208